.github/stale.yml

```diff
@@ -0,0 +1,17 @@
+# Number of days of inactivity before an issue becomes stale
+daysUntilStale: 90
+# Number of days of inactivity before a stale issue is closed
+daysUntilClose: 15
+# Issues with these labels will never be considered stale
+exemptLabels:
+  - "🐞 bug"
+# Label to use when marking an issue as stale
+staleLabel: "🚫 won't fix"
```

Comment:
Won't fix and stale are not the same. I'd like to have a label that we can filter after and use to just filter stale issues.
```
staleLabel: "🏚 stale"
```

Reply:
```suggestion
staleLabel: "🏚 stale"
```